CODEOWNERS

```diff
@@ -0,0 +1,82 @@
+# Catch-all
+* @tgraf @manalibhutiyani
+
+# BPF Data Path
+bpf/* @joestringer @borkmann @jrfastab @manalibhutiyani
+pkg/maps/* @joestringer @borkmann @jrfastab @manalibhutiyani
+pkg/bpf/* @joestringer @borkmann @jrfastab @manalibhutiyani
+# TODO Where are "KProxy kernel" and "XDP Prefilter" parts?
+
+# Agent -> Kernel interaction
+# TODO No idea where is it
+# @scanf @borkmann
+
+# CI
+tests/* @ianvernon @eloycoto
+Jenkinsfile @ianvernon @eloycoto
+
+# KVStore
+pkg/kvstore/* @ianvernon @aanm @tgraf @rlenglet
+
+# K8s integration, K8s CNI plugin
+daemon/k8s_watcher.go @aanm @nebril
+pkg/k8s/* @aanm @nebril
+plugins/cilium-cni/* @aanm @nebril
+
+# Docker plugin
+plugins/cilium-docker/* @nebril @manalibhutiyani
+
+#TODO: If mesos plugin is created: @nebri @manalibhutiyani @aanm
+
+# Policy behaviour
+pkg/policy @nebril @jrajahalme @manalibhutiyani
+
+# Agent API
+api/* @rlenglet @ianvernon @manalibhutiyani
```

Comment:
I've informed @rlenglet of this offline on Slack - we need to wait for his approval for this as we volunteered his name :-)

Reply:
LGTM